src/WORKSPACE

```diff
@@ -99,14 +99,10 @@ new_local_repository(
     path = "external/qt",  # This directory is symlinked in the setup script
 )
 
-http_archive(
-    name = "llvm_clang",
-    build_file = "@//external:clang.BUILD",
-    sha256 = "a23b082b30c128c9831dbdd96edad26b43f56624d0ad0ea9edec506f5385038d",
-    strip_prefix = "clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-18.04",
-    urls = [
-        "http://releases.llvm.org/9.0.0/clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz",
-    ],
+new_local_repository(
+    name = "linux_gcc_7_4_0",
```

Comment:
Bit of a nitpick. My understanding is that this toolchain is effectively saying "use system gcc". If that's the case should this be named as such? I realize we are making the assumption development takes place on Ubuntu 18 which has gcc 7.4.0 as default, and we install gcc-7 in the setup scripts just in case.

This just might be confusing for people trying to upgrade later?

Reply:
Good point. Changed `linux_gcc_7_4_0` to `linux_gcc` everywhere.